app/global-translations/locale-en.json

```diff
@@ -2494,6 +2494,8 @@
     "validation.msg.hook.multiple.non.web.template.hooks.not.supported": "Multiple hooks for non-web dedicated services not allowed.",
     "error.msg.duplicate.entity.mapping":"Duplicate mapping, EntityMapping already exist",
     "error.msg.invalid.endDate":"End date cannot be before start date",
+    "error.msg.password.reset.days.value.must.be.greater.than.zero" : "Force Password Reset days value must be greater than 0.",
+
     "#tool tip for global configuration":"..",
```

Comment:
Could you please modify the error message to 

For enabling 'Force Password Reset Days' configuration , the value (number of days after which a user is forced to reset his password) must be set to a number greater than 0